error during compilation

Hello.
I try to make citus from source.
I set  PG_CONFIG.
When I do make, I have next error:
What is it? Whot do I not right?


In file included from /home/worker1/soft/pg_sql-worker-1/include/postgresql/server/postgres.h:47:0,
                 from master/master_create_shards.c:13:
master/master_create_shards.c: In function ‘master_create_worker_shards’:
/home/worker1/soft/citus/src/include/distributed/master_metadata_utility.h:25:37: error: invalid suffix "ULLL" on integer constant
 #define HASH_TOKEN_COUNT INT64CONST(4294967296UL)
                                     ^
/home/worker1/soft/pg_sql-worker-1/include/postgresql/server/c.h:280:33: note: in definition of macro ‘INT64CONST’
 #define INT64CONST(x)  ((int64) x##LL)
                                 ^
master/master_create_shards.c:118:23: note: in expansion of macro ‘HASH_TOKEN_COUNT’
  hashTokenIncrement = HASH_TOKEN_COUNT / shardCount;
                       ^
make[1]: *** [master/master_create_shards.o] Error 1

I use Ubuntu (Distributor ID: Ubuntu,Description:    Ubuntu 14.04.4 LTS, Release:        14.04, Codename:       trusty)
Postgresql (9.5.1) from github.
 git clone https://github.com/citusdata/citus.git
cd citus
PG_CONFIG=/home/alex/soft/pg_sql-citus1/bin/pg_config
./configure
(all ok)
make -s 
and the error is 

In file included from /home/alex/soft/pg_sql-citus1/include/postgresql/server/postgres.h:47:0,
                 from master/master_create_shards.c:13:
master/master_create_shards.c: In function ─≤master_create_worker_shards─≥:
/home/alex/soft/citus/src/include/distributed/master_metadata_utility.h:25:37: error: invalid suffix "ULLL" on integer constant
 #define HASH_TOKEN_COUNT INT64CONST(4294967296UL)
                                     ^
/home/alex/soft/pg_sql-citus1/include/postgresql/server/c.h:280:33: note: in definition of macro ─≤INT64CONST─≥
 #define INT64CONST(x)  ((int64) x##LL)
                                 ^
master/master_create_shards.c:118:23: note: in expansion of macro ─≤HASH_TOKEN_COUNT─≥
  hashTokenIncrement = HASH_TOKEN_COUNT / shardCount;
                       ^
make[1]: *** [master/master_create_shards.o] Error 1
make: *** [extension] Error 2

--- a/src/backend/distributed/master/master_metadata_utility.c
+++ b/src/backend/distributed/master/master_metadata_utility.c
@@ -337,7 +337,7 @@ TupleToShardPlacement(TupleDesc tupleDescriptor, HeapTuple heapTuple)
 	shardPlacement->shardLength = DatumGetInt64(shardLength);
 	shardPlacement->shardState = DatumGetUInt32(shardState);
 	shardPlacement->nodeName = TextDatumGetCString(nodeName);
-	shardPlacement->nodePort = DatumGetUInt32(nodePort);
+	shardPlacement->nodePort = DatumGetInt64(nodePort);
 
 	return shardPlacement;
 }
@@ -420,7 +420,7 @@ InsertShardPlacementRow(uint64 shardId, char shardState, uint64 shardLength,
 	values[Anum_pg_dist_shard_placement_shardstate - 1] = CharGetDatum(shardState);
 	values[Anum_pg_dist_shard_placement_shardlength - 1] = Int64GetDatum(shardLength);
 	values[Anum_pg_dist_shard_placement_nodename - 1] = CStringGetTextDatum(nodeName);
-	values[Anum_pg_dist_shard_placement_nodeport - 1] = UInt32GetDatum(nodePort);
+	values[Anum_pg_dist_shard_placement_nodeport - 1] = Int64GetDatum(nodePort);
 
 	/* open shard placement relation and insert new tuple */
 	pgDistShardPlacement = heap_open(DistShardPlacementRelationId(), RowExclusiveLock);

